### ISSUE TITLE
Does not show the view

### ISSUE THREAD

**Issue by sadegh1998** [opensource-emr/Telemedicine#124]
when iWhen I run the project, a white screen is displayed and there is no login panel. In the view index the paths clientapp/runtime.js etc. are not found.
How can I run the project?

**Comment by Greatworkmayank** [comment]
yes im also having same problem i cant find the solution for that please mention here if you found any
